engines/sci/engine/script_patches.cpp

```diff
@@ -18435,6 +18440,43 @@ void ScriptPatcher::applyPatch(const SciScriptPatcherEntry *patchEntry, SciSpan<
 	}
 }
 
+
+// TODO: suggest automatic installation
+void ScriptPatcher::suggestDownloadGK2SubTitlesPatch() {
+	if (ConfMan.getBool("subtitles")) {
+
+		const char *altButton;
+		Common::String downloadMessage;
+
+		if (g_system->hasFeature(OSystem::kFeatureOpenUrl)) {
+			altButton = _("Download patch");
+			downloadMessage = _("(or click 'Download patch' button. But note - it only downloads, you will have to continue from there)\n");
+		} else {
+			altButton = nullptr;
+			downloadMessage = "";
+		}
+
+		GUI::MessageDialog dialog(_("GK2 has a fan made subtitles, available thanks to the good persons at SierraHelp.\n\n"
+			"Installation:\n"
+			"- download http://www.sierrahelp.com/Files/Patches/GabrielKnight/GK2Subtitles.zip\n" +
+			downloadMessage +
+			"- extract zip file\n"
+			"- no need to run the .exe file\n"
+			"- extract the .exe file with 7-zip (don't worry regarding resource.cfg, you can just replace it, it's not used anyway)\n"
+			"- copy the content of GK2Subtitles\\SUBPATCH to:\n"
+			"  = if your game directory has a PATCHES directory, to there\n"
+			"  = if not, to the game directory\n"
```

Comment:
It's easier to just copy the contents into the PATCHES directory, and create it if it doesn't exist

```suggestion
			"- create a PATCHES subdirectory inside your GK2 directory\n"
			"- copy the content of GK2Subtitles\\SUBPATCH to the PATCHES subdirectory\n"
```